cosmos/precompile/staking/staking.go

```diff
@@ -57,6 +63,7 @@ func NewPrecompileContract(sk *stakingkeeper.Keeper) *Contract {
 			generated.StakingModuleMetaData.ABI,
 			cosmlib.AccAddressToEthAddress(authtypes.NewModuleAddress(stakingtypes.ModuleName)),
 		),
+		ac:        sk,
```

Comment:
storing 3 pointers to the same thing now kek, i guess its fine tho

Reply:
yea but no access cause interfaces

Reply:
yeah no stress, it's kinda ugly no matter how u slice it